presto-hive/src/test/java/io/prestosql/plugin/hive/benchmark/BenchmarkHiveFileFormatUtil.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.hive.benchmark;
+
+import com.google.common.collect.ImmutableList;
+import io.prestosql.spi.Page;
+import io.prestosql.spi.type.Type;
+
+import java.util.List;
+
+public class BenchmarkHiveFileFormatUtil
+{
+    private BenchmarkHiveFileFormatUtil() {}
+
+    protected static class TestData
```

Comment:
Since we're using `TestData` in multiple independent classes, I thought it might be better to extract it out. I hadn't thought about making it top-level, but it makes sense with the name you suggested, rather than putting in a random util class.